### PR TITLE
refactor locals folders

### PR DESCRIPTION
**Pull request by gmichokostas**
Hi @marcaltmann I opend this pull request because I saw the deprecation message on README.txt,
I' ve notice that there are on many views hard coded strings and I would like to add them on a local file. Shall I do all the work on this branch and you can merge it later? 
Thanks

### CODE REVIEW

**Comment by marcaltmann** [comment]
Hi @gmichokostas, I haven't really familiarized myself with the internationalization guidelines yet, but I think they should be okay. So if you like you can start. Maybe its better for your own workflow if you do the work on your feature_refactor_translations branch. On our side it doesn't matter much. Also, if you like, I can pull in parts of your work in steps. It does not have to be done all at once.

**Comment by gmichokostas** [comment]
please don't merge this, I needs rebase.

File: app/views/addresses/edit.html.slim [hunk 1]
```diff
@@ -1,3 +1,3 @@
 .Address-form-container
-  h1 Adresse bearbeiten
+  h1= t('address.actions.edit')
```
Comment (marcaltmann) [inline app/views/addresses/edit.html.slim]: Hi @gmichokostas, would you please reference the locales in the views according to "lazy loading"? In this case you would need to use t('.heading') and in the YAML file just put the string under 'de.addresses.edit.heading'. And you would rename the YAML file to "views/addresses/de.yml".
The translations for the welcome#index view are already in this lazy loading format.
Thanks!

**Comment by marcaltmann** [comment]
Hi @gmichokostas by the way: Actually we haven't really decided wether we squash commits or not. So you can do it how it works best for you.

**Comment by gmichokostas** [comment]
Hey @marcaltmann I left the commits unsquashed as you said, but I haven't rebase this branch on the latest upstream/develop. Do you think is OK?

**Comment by marcaltmann** [comment]
Hi @gmichokostas,
there are two issues that I see with your PR:
* I would not move all the files into the views directory yet, only those that are completely transformed to "lazy lookup" style. Also, I would not delete the README until the transformation is over (sorry for pointing this out so late).
* The translation "Neue Adresse hinzufügen" does not work anymore (screenshot attached), because you changed the partial that calls it to "lazy lookup" without putting the string in a file that the partial can find. 

As soon as you're done, you can either rebase the branch to the latest upstream (I prefer it) or I can do it when merging it. (BTW: If you have the time you can also drop by our office, if you'd like...)
translation_error

**Comment by gmichokostas** [comment]
Hi @marcaltmann I am going to close this pull request and open a new one with different branch name. I think is better to pull smaller parts of the translation process so we don't mess it up.
p.s. Thanks for the invitation sure I am going to pass by and meet you guys sometime soon.

**Pull request closed without merging**